### PR TITLE
Apply resource naming convention updates

### LINKED ISSUE #38: Restructure API model to follow resource naming conventions
- routes should reflect collections
- verb should specify CRUD operation
- query parameters should be utilized for restrictions
- body utilized when inserting/mutating data

https://restfulapi.net/resource-naming/

### PR DESCRIPTION
**Pull request by guzman-raphael**
### Added
- Local database instance pre-populated with sample data for `dev` Docker Compose environment. PR #99
- Capability to insert multiple, update multiple, and delete multiple. PR #99
- Allow dependency restriction to include secondary attributes from parent table. PR #99

### Changed
- Rename service `pharus` to `pharus-docs` in `docs` Docker Compose environment to allow simulataneous development. PR #99
- Update NGINX reverse proxy image reference. PR #99
- Refactored API design to align with common REST resource naming convention. (#38) PR #99
- Hide classes and methods that are internal and subject to change. PR #99

### Removed
- `InvalidDeleteRequest` exception is no longer available as it is now allowed to delete more than 1 record at a time. PR #99

Fix #38

### CODE REVIEW

File: pharus/interface.py [hunk 1]
```diff
@@ -219,84 +194,87 @@ def get_table_attributes(jwt_payload: dict, schema_name: str, table_name: str) -
         :type schema_name: str
         :param table_name: Table name under the given schema; must be in camel case
         :type table_name: str
-        :return: Dict with keys ``primary_attributes``, ``secondary_attributes`` containing a
+        :return: Dict with keys ``attribute_headers`` and ``attributes`` containing
+            ``primary_attributes``, ``secondary_attributes`` which each contain a
```
Comment (Synicix) [inline pharus/interface.py:198]: I think this should be primary and secondary without the attributes

**Review by Synicix — APPROVED** [review]
Just need one minor doc change

File: pharus/interface.py (hunk 1 above)
Comment (guzman-raphael) [reply pharus/interface.py:198]: @Synicix Not near my machine right now but if you will propose the change I can accept it.

Comment (Synicix) [reply pharus/interface.py:198]: Suggestion added

Comment (guzman-raphael) [reply pharus/interface.py]: @Synicix Thanks, approved and included it.

**Review by Synicix — APPROVED** [review]

**Pull request merged**